bigtable/CHANGELOG.md

```diff
@@ -4,6 +4,66 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.30.0
```

Comment:
Can you categorize these changes into the headings below?

Reply:
Yup.  I'll mark this PR as a Work In Progress / WIP, and then get to work on making this changelog easier to grok.

Reply:
Also needs to include PR #5790, which I just merged.